### PR TITLE
Make the target path of the superset logo configuable

### PR DESCRIPTION
**Pull request by xtinec**
There is a need to have the target path of the logo go to different places in different installations of superset. The PR makes the target path (after `/superset`) configurable via the LOGO_TARGET_PATH configuration key and sets the default to the existing `/profile/{{ current_user.username }}/`

### CODE REVIEW

**Comment by betodealmeida** [comment]
LGTM, I would just add an entry in `config.py` or in the documentation describing the feature, so users are aware of the feature.

Maybe add this to `config.py`:

```python
# bla bla bla
LOGO_TARGET_PATH = None
```

And then update the template to:

```python
logo_target_path = appbuilder.app.config.get('LOGO_TARGET_PATH') or '/profile/{}/'.format(current_user.username)
```

**Comment by xtinec** [comment]
`set logo_target_path = appbuilder.app.config.get('LOGO_TARGET_PATH', '/profile/{}/'.format(current_user.username))` does the same thing as the `or` suggested above, right?

Also, do we need the `LOGO_TARGET_PATH = None` in `config.py` as the fallback will always kick in when `LOGO_TARGET_PATH` is not defined?

Good call on the doc. I'll update the readme.

**Comment by betodealmeida** [comment]
There's a subtle difference. If you have in `config.py`:

```python
LOGO_TARGET_PATH = None
```

Then:

```python
>>> print(config.get('LOGO_TARGET_PATH', 'foo'))
None
>>> print(config.get('LOGO_TARGET_PATH') or 'foo')
foo
```

Because `.get` only checks if the key exists, not if the associated value is truthy.

I think adding the line in `config.py` is useful because it shows users that they can change the logo.

File: superset/config.py [hunk 1]
```diff
@@ -112,6 +112,10 @@
 APP_ICON = '/static/assets/images/superset-logo@2x.png'
 APP_ICON_WIDTH = 126
 
+# Uncomment to specify where clicking the logo would take the user
```
Comment (betodealmeida) [inline superset/config.py]: Nit: replace "uncomment" with "set this"

**Review by betodealmeida — APPROVED** [review]
LGTM, feel free to squash and merge once the tests pass. Note that I commented on a nit in the `config.py`.

**Pull request merged**